pipeline-runner/R/gem2s-2-load_user_files.R

```diff
@@ -45,6 +46,64 @@ load_user_files <- function(input, pipeline_config, prev_out, input_dir = INPUT_
   return(res)
 }
 
+#' Read Parse files
+#'
+#' @param config
+#' @param input_dir
+#'
+#' @return
+#' @export
+#'
+#' @examples
+read_parse_files <- function(config, input_dir) {
+  counts_list <- list()
+  annot_list <- list()
+  feature_types_list <- list()
+
+  samples <- config$samples
+
+  for (sample in samples) {
+    sample_dir <- file.path(input_dir, sample)
+    sample_fpaths <- list.files(sample_dir)
+    annot_fpath <- file.path(sample_dir, "all_genes.csv.gz")
+
+    message("\nSample --> ", sample)
+    message(
+      "Reading files from ",
+      sample_dir,
+      " --> ",
+      paste(sample_fpaths, collapse = " - ")
+    )
+
+    annotations <- read_parse_annotations(annot_fpath, sample)
+
+    mtx <- file.path(sample_dir, "DGE.mtx.gz")
+    cells <- file.path(sample_dir, "cell_metadata.csv.gz")
+    features <- file.path(sample_dir, "all_genes.csv.gz")
```

Comment:
these as well. cells -> barcodes for example, to be consistent with naming in the other read_* functions